addons/koji/common/src/main/java/org/commonjava/indy/koji/content/KojiContentManagerDecorator.java

```diff
@@ -307,8 +305,8 @@ public Transfer getTransfer( final ArtifactStore store, final String path, final
             return kojiClient.withKojiSession( ( session ) -> {
                 List<KojiBuildInfo> builds = kojiClient.listBuildsContaining( artifactRef, session );
 
-                Collections.sort( builds, ( build1, build2 ) -> build1.getCreationTime()
-                                                                      .compareTo( build2.getCreationTime() ) );
+                Collections.sort( builds, ( build1, build2 ) -> build2.getCreationTime()
+                                                                      .compareTo( build1.getCreationTime() ) );
```

Comment:
I think usually the last build of a GAV is published to MRRC, so using it as authority source makes all but last builds skipped. And every check sends a listArchives request to koji, so checking the latest build as the first one avoids unnecessary requests for the first N-1 builds if there is more than one.